app_test.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"context"
 	"fmt"
+	"go.ytsaurus.tech/library/go/ptr"
```

Comment:
> File is not `goimports`-ed with -local go.ytsaurus.tech (goimports)

actually I think we should remove that rule from linter config, because this library is external to this project.